services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/feature/AbstractFeature.java

```diff
@@ -20,10 +20,11 @@
 import net.floodlightcontroller.core.IOFSwitch;
 
 import java.util.Optional;
+import java.util.regex.Pattern;
 
 public abstract class AbstractFeature {
     protected static final String MANUFACTURER_NICIRA = "Nicira, Inc.";
-    protected static final String MANUFACTURER_NOVIFLOW = "NoviFlow Inc";
+    protected static final Pattern MANUFACTURER_NOVIFLOW = Pattern.compile("^NW\\d{3}\\.\\d+\\.\\d+$");
```

Comment:
I think we need unit tests for this regexp

Reply:
This is definetelly not "MANUFACTURER_NOVIFLOW". Keep this regexp on BfdFeature "level" and drop this constant if it is not used any more.